.github/workflows/build.yml

```diff
@@ -28,6 +28,9 @@ jobs:
        run: |
          pip install poetry
          poetry install
+         curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y --default-toolchain stable
```

Comment:
We can use some pre-made GitHub Actions here for setting up [rust toolchain](https://github.com/marketplace/actions/rust-toolchain) or even [mdbook](https://github.com/marketplace/actions/mdbook-action) for better readability. But this line of code do the job and there is no need to modify anything now.

Reply:
Good point. But, installing `Rust` + `mdBook` sounds overkill for most of pyserde developers 🤔  I will look for a way to install mdBook as a single binary when I have time.